plugins/defaultplugins/ifplugin/vppcalls/vrf_vppcalls.go

```diff
@@ -115,14 +115,10 @@ func dumpVrfTables(vppChan VPPChannel) (map[uint32][]*ip.IPFibDetails, error) {
 	return fibs, nil
 }
 
-func vppAddDelIPTable(tableID uint32, vppChan VPPChannel, delete bool) error {
+func vppAddDelIPTable(tableID uint32, vppChan VPPChannel) error {
```

Comment:
This function was intended to be used for add or del, hence the name. Even though for now deleting was needed yet.

Reply:
I got that, however, the function is not exported and 'delete' was used only once with the false value. It means that the old code (line 121 - 125) could not be tested calling exported functions only. 
I may rename it to vppAddIPTable. Delete can be easily added if needed.